RecoBTag/FeatureTools/plugins/DeepBoostedJetTagInfoProducer.cc

```diff
@@ -826,7 +832,7 @@ void DeepBoostedJetTagInfoProducer::fillParticleFeatures(DeepBoostedJetFeatures
                                              pv_ass.key());
             candidate.setAssociationQuality(pat::PackedCandidate::PVAssociationQuality(
                 btagbtvdeep::vtx_ass_from_pfcand(*reco_cand, pv_ass_quality, pv_ass)));
-            candidate.setCovarianceVersion(0);
+            candidate.setCovarianceVersion(covarianceVersion_);
```

Comment:
mmmh, this makes me realize that this was hard-coded for phase-0... 
Shouldn't this be dealt with via era modifier as elsewhere in cmssw?

https://github.com/cms-sw/cmssw/blob/4a6d7e831d09c01a92d2381c319f1424a524e651/PhysicsTools/PatAlgos/python/slimming/lostTracks_cfi.py#L29-L30

Reply:
Exactly, it is HLT configuration which is not era dependent in the same sense as offline. I do not know if it was intention of developers of the module to use phase0 pixel setting - it is question to BTV people.
This development is to help dealing with those settings using configuration parameters (maybe it can be checked before 2024 data-taking if usage of 1 is fine) without need to modify hard-coded vales.

Reply:
Apparently, this was a mistake by the original author. He mistook the default value in the producer of `0` as the default. It clearly should be handled as discussed above. I did a study on 70k jets, and the current generation of taggers is unaffected by this choice, we can change it.

Reply:
hi @SWuchterl 

> Apparently, this was a mistake by the original author. He mistook the default value in the producer of 0 as the default. It clearly should be handled as discussed above

thanks for cross-checking! Once this PR is accepted and the new configuration is parsed, can you ask your HLT L3s to open a JIRA ticket to [CMSHLT](https://its.cern.ch/jira/projects/CMSHLT/issues) in order to change the configuration in the HLT menu for 2024 (assuming you want to move to the packing scheme for phase-1) ?